<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin</class>
 <widget class="QMainWindow" name="MainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Moswar bot beta</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/favicon.png</normaloff>:/images/favicon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QToolButton" name="tbStart">
        <property name="text">
         <string>Start</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/start.png</normaloff>:/images/start.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbStop">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="toolbar" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="tbAtack">
           <property name="toolTip">
            <string>Атака</string>
           </property>
           <property name="text">
            <string>Atack</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/atack.png</normaloff>:/images/atack.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbRat">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Охота на крыс</string>
           </property>
           <property name="text">
            <string>Rat</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/rat.png</normaloff>:/images/rat.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbDig">
           <property name="toolTip">
            <string>Копать в метро</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/kirka.png</normaloff>:/images/kirka.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbOil">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Нефтепровод</string>
           </property>
           <property name="text">
            <string>Neft</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/neft.png</normaloff>:/images/neft.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbThimble">
           <property name="toolTip">
            <string>Игра с моней</string>
           </property>
           <property name="text">
            <string>Thimble</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/ruda.png</normaloff>:/images/ruda.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbPetrik">
           <property name="toolTip">
            <string>Варка петриков</string>
           </property>
           <property name="text">
            <string>Petrik</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/petrik.png</normaloff>:/images/petrik.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbGipsy">
           <property name="text">
            <string>Gipsy</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/gold.png</normaloff>:/images/gold.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbBaraban">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/baraban.png</normaloff>:/images/baraban.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbSellLot">
           <property name="toolTip">
            <string>Продажа коктейлей</string>
           </property>
           <property name="text">
            <string>Lots</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/cocktail.png</normaloff>:/images/cocktail.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Browser</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWebView" name="browser">
          <property name="url">
           <url>
            <string>about:blank</string>
           </url>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Options</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>Закоулки</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QCheckBox" name="cbAtack">
                <property name="text">
                 <string>Атаковать</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cbAtackType"/>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>или</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cbAType2"/>
              </item>
              <item>
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Уровень от</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sbMinLev"/>
              </item>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>до</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sbMaxLev"/>
              </item>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>и статы</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="sbStatCheck">
                <property name="maximum">
                 <double>2.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>от игрока</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cbNPCheck">
                <property name="text">
                 <string>не учитывать NPC</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_9">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_6">
           <attribute name="title">
            <string>Завод</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QCheckBox" name="cbPetrik">
              <property name="text">
               <string>Варить петрики</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>552</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>304</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>Метро</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QCheckBox" name="cbMonia">
                <property name="text">
                 <string>Играть с Моней за</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sbMoniaCoins">
                <property name="minimum">
                 <number>6000</number>
                </property>
                <property name="maximum">
                 <number>99999</number>
                </property>
                <property name="value">
                 <number>30000</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="resources.qrc">:/images/money.png</pixmap>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cbPlayTickets">
                <property name="text">
                 <string>за билеты</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cbBuyTickets">
                <property name="text">
                 <string>покупать их</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cbTicketStars">
                <property name="text">
                 <string>за звезды</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QCheckBox" name="cbRatHunt">
                <property name="text">
                 <string>Охотиться на крыс</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QCheckBox" name="cbDigger">
                <property name="text">
                 <string>Копать туннель</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cbDigRat">
                <property name="text">
                 <string>бить крыс</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_7">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_7">
           <attribute name="title">
            <string>Разное</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QCheckBox" name="cbRoll">
                <property name="text">
                 <string>Крутить барабан на все и покупать</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sbCaps">
                <property name="minimum">
                 <number>0</number>
                </property>
                <property name="maximum">
                 <number>20</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>фишек</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_8">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>303</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pbOptSave">
            <property name="text">
             <string>Сохранить</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTextBrowser" name="log">
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="openLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,10">
      <item>
       <widget class="QProgressBar" name="progress">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>732</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
